File: TP1/src/classes/position.py
class Position(object):
    def __init__(self, x, y):
        self.x = x
        self.y = y

    def get_X(self):
        return self.x

    def get_Y(self):
        return self.y

    def __str__(self):
        return "(" + str(self.x) + ", " + str(self.y) + ")"

File: src/main/java/tp1/classes/Position.java
package tp1.classes;

public class Position {

	private float x;
	private float y;

	public Position(float x, float y) {
		this.y = y;
		this.x = x;
	}

	public float getX() {
		return x;
	}

	public void setX(float x) {
		this.x = x;
	}

	public float getY() {
		return y;
	}

	public void setY(float y) {
		this.y = y;
	}
	
	public String print() {
		return x+"\t"+y;
	}
	
	public String toString() {
		return "("+x+", "+y+")";
	}
}
